pkg/commands/workload_apply_test.go

```diff
@@ -3745,10 +3906,14 @@ To get status: "tanzu apps workload get spring-petclinic"
 `,
 		},
 		{
-			Name: "update git fields",
+			Name: "update - do not default type in existing workload",
```

Comment:
@shaheerkootteeri added new test case to avoid adding `type` in existing workload